### ISSUE TITLE
Remove deprecated constants

### ISSUE THREAD

**Issue by Jeroen12345** [drush-ops/drush#1183]
In Drupal 8, the following constants in bootstrap.inc are deprecated and the usage in drush should be removed: 

* DRUPAL_BOOTSTRAP_CONFIGURATION
* DRUPAL_BOOTSTRAP_KERNEL
* DRUPAL_BOOTSTRAP_CODE
* DRUPAL_BOOTSTRAP_FULL

Issue on drupal.org to remove this constants is: https://www.drupal.org/node/2347625.

**Comment by weitzman** [comment]
See function [drupal8_bootstrap](http://api.drush.org/api/drush/lib!Drush!Boot!bootstrap.inc/function/drupal8_bootstrap/master)

**Comment by weitzman** [comment]
OK. I think we need to wait for that D8 patch to land.

**Comment by Berdir** [comment]
I don't think so, because it's just removed, so we have to find a different way to do it anyway. Which I tried in https://github.com/drush-ops/drush/pull/1230
